gramps_gedcom7/event.py

```diff
@@ -42,9 +42,37 @@ def handle_event(
                 event.set_type(EventType(child.value))
         elif child.tag == g7const.RESN:
             util.set_privacy_on_object(resn_structure=child, obj=event)
+        elif child.tag == g7const.PHON:
+            assert isinstance(child.value, str), "Expected value to be a string"
+            attr = Attribute()
+            attr.set_type(AttributeType.CUSTOM)
+            attr.get_type().string = "Phone"
+            attr.set_value(child.value)
```

Comment:
Nitpicking now - Gramps types are pretty horrible to work with IMO - but the easier (and possibly more readable) way to instantiate a custom attribute type would be to simply do

```python
attr.set_type(AttributeType("Phone"))
```

and analogously below, saving 2 lines each. This works because of the singledispatch for strings in `GrampsType.set` which automatically makes this a custom type.
https://github.com/gramps-project/gramps/blob/master/gramps/gen/lib/grampstype.py#L199-L206